examples/burial-form/ClaimantContactInformation.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {AddressField, EmailField, Page, TextField} from '@department-of-veterans-affairs/va-forms-system-core';
+import {useFormikContext} from "formik";
+import PhoneField from "@department-of-veterans-affairs/va-forms-system-core/form-builder/PhoneField";
+
+export default function ClaimantContactInformation(props) {
+  const formikContext = useFormikContext();
+
+  formikContext?.values?.relationship?.isEntity = true;
+
+  return (
+    <>
+      <Page {...props} nextPage="/review-and-submit" prevPage="/benefits/plot-allowance">
+        {formikContext?.values?.relationship?.isEntity && (
```

Comment:
Does this need to be conditionally rendered? I could be missing something but it seems like `isEntity` is being assigned the value of `true` on line 9, so wouldn't this always just be rendered?

Reply:
Those two fields only can render if `isEntity = true`, however the code setting `isEntity = true` was just for testing. I have removed that line after making sure it works for both cases